crypto/fipsmodule/ml_kem/importer.sh

```diff
@@ -94,12 +102,29 @@ cp $TMP/.clang-format $SRC
 # The static simplification is not necessary, but improves readability
 # by removing directives related to the FIPS-202 backend and the x86_64
 # arithmetic backend that are not yet imported.
+# Moreover, exclude POLY_COMPRESS/DECOMPRESS functions from the x86 backend.
 unifdef -DMLK_CONFIG_FIPS202_CUSTOM_HEADER                             \
         -UMLK_CONFIG_USE_NATIVE_BACKEND_FIPS202                        \
-        -UMLK_SYS_X86_64                                               \
         $TMP/mlkem/mlkem_native.c                                      \
         > $SRC/mlkem_native_bcm.c
 
+if [[ "$(uname)" == "Darwin" ]]; then
+  SED_I=(-i "")
+else
+  SED_I=(-i)
+fi
+
+# Exclude POLY_COMPRESS/DECOMPRESS functions from the x86 backend for now.
```

Comment:
Why?

Reply:
Because those are implemented in C using the AVX2 C intrinsics which we can't handle in aws-lc. The plan is to implement the functions in assembly and then import to aws-lc, but we'll do that only after we formally verify the existing assembly (hopefully by the end of year).